llvm/lib/Target/Parasol/ParasolISelDAGToDAG.cpp

```diff
@@ -37,11 +37,80 @@ void ParasolDAGToDAGISel::Select(SDNode *Node) {
 
   // Instruction Selection not handled by the auto-generated tablegen selection
   // should be handled here.
-  switch (Opcode) {
+  switch (Node->getOpcode()) {
   default:
     break;
+    // case ISD::FrameIndex: {
```

Comment:
Remove. We moved to using a ComplexPat to rewrite FrameIndex nodes as ADDri. Then during frame elimination, we write the final stack/frame pointer and offset add+loadI sequence.